joomla/joomla-cms
# [3.10] Show the target version to the pre upgrade checker

Pull Request for Issue #31292

### Summary of Changes

Show the target version to the pre upgeade checker

### Testing Instructions

- install 3.10
- set the custom update server to: `https://update.joomla.org/core/test/310to4_list.xml`
- check the pre upgrade checker

### Actual result BEFORE applying this Pull Request

![image](https://user-images.githubusercontent.com/2596554/105409477-ccdaa980-5c30-11eb-94d8-2ab52d7b3c6f.png)

### Expected result AFTER applying this Pull Request

![image](https://user-images.githubusercontent.com/2596554/105409342-9c930b00-5c30-11eb-8dbb-4992887d4205.png)


### Documentation Changes Required

none
